pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/index/TextIndexConfig.java

```diff
@@ -75,6 +76,7 @@ public class TextIndexConfig extends IndexConfig {
   private final int _luceneNRTCachingDirectoryMaxBufferSizeMB;
   private final boolean _useLogByteSizeMergePolicy;
   private final DocIdTranslatorMode _docIdTranslatorMode;
+  private final boolean _caseSensitiveIndex;
```

Comment:
I feel `caseSensitive` is more concise. Does case sensitive apply to anything other than index?